commands/utils.go

```diff
@@ -15,33 +12,59 @@ import (
 const (
 	httpContentType     = "Content-Type"
 	httpContentTypeJSON = "application/json"
+	httpContentTypeXML  = "application/xml"
 )
 
+type flagValueProvider interface {
+	GetStringFlagValue(flagName string) string
+}
+
+type serviceHelper interface {
+	GetConfig(serverID string, excludeRefreshableTokens bool) (*config.ArtifactoryDetails, error)
+	CreateInitialRefreshableTokensIfNeeded(artifactoryDetails *config.ArtifactoryDetails) error
+}
+
 // Returns the Artifactory Details of the provided server-id, or the default one.
-func getRtDetails(c *components.Context) (*config.ArtifactoryDetails, error) {
-	details, err := commands.GetConfig(c.GetStringFlagValue(serverID), false)
+func getRtDetails(flagProvider flagValueProvider, configHelper serviceHelper) (*config.ArtifactoryDetails, error) {
+	serverID := flagProvider.GetStringFlagValue(serverIDFlag)
+	return buildRtDetailsFromServerID(serverID, configHelper)
+}
+
+// Returns the Artifactory Details of the target-server-id, or JFrog support logs configured ArtifactoryDetails.
+func getTargetDetails(flagProvider flagValueProvider, configProvider serviceHelper,
+	conf *supportBundleCommandConfiguration) (*config.ArtifactoryDetails, error) {
+	serverID := flagProvider.GetStringFlagValue(targetServerID)
+	if serverID == jfrogSupportLogsArtifactory {
```

Comment:
Maybe compare with `""`?

Reply:
Yes, I thought about that. It is probably better.